docs/services.md

```diff
@@ -579,26 +579,18 @@ services.clightning = {
 
 Please have a look at the module for a plugin (e.g. [prometheus.nix](../modules/clightning-plugins/prometheus.nix)) to learn its configuration options.
 
-### Trustedcoin hints
-The [trustedcoin](https://github.com/nbd-wtf/trustedcoin) plugin use a Tor
-proxy for all of its external connections by default. That's why you can
-sometimes face issues with your connections to esploras getting blocked.
-
-An example of clightning log error output in a case your connections are getting blocked:
+### Trustedcoin
+When `services.clightning.tor.proxy` is enabled, [trustedcoin](https://github.com/nbd-wtf/trustedcoin)
+also uses Tor for all external connections by default.
+In this case, connections to block explorers can sometimes get blocked.
 
+An example of clightning log error output when connections are getting blocked:
 ```
 lightningd[5138]: plugin-trustedcoin estimatefees error: https://blockstream.info/api error: 403 Forbidden
+lightningd[4933]: plugin-trustedcoin getblock error: got something that isn't a block hash: <html><head>...
 ```
 
-```
-lightningd[4933]: plugin-trustedcoin getblock error: got something that isn't a block hash: <html><head>
-lightningd[4933]: <meta http-equiv="content-type" content="text/html;
-```
-
-If you face these issues and you still need to use trustedcoin, use can disable
-clightning's tor hardening by setting this option in your `configuration.nix`
-file:
-
-```
-services.clightning.tor.enforce = false;
+To work around this and connect via clearnet instead, set this option:
+```nix
+services.clightning.plugins.trustedcoin.enableTor = false;
```

Comment:
This does not look like an improvement because it suggests that Tor is only disabled for trustedcoin and not for all of clightning.

Reply:
No, the point of `trustedcoin.enableTor` is to disable Tor only for trustedcoin, but not for clightning.

`trustedcoin.enableTor` sets `clightning.tor.enforce == false`.
This is required to enable trustedcoin clearnet connections. (trustedcoin runs in the clightning service namespace.)
But the value of `clightning.tor.proxy`, which enables Tor for clightning, is unaffected.

Reply:
Right, this just sets option just sets `clightning.tor.enforce`.